pkgs/servers/owncast/default.nix

```
@@ -1,43 +1,31 @@
-{ lib, buildGoModule, fetchFromGitHub, nixosTests, bash, which, ffmpeg, makeWrapper, coreutils, ... }:
-
-buildGoModule rec {
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+, ffmpeg
+, makeWrapper
```

```suggestion
, makeBinaryWrapper
```

Use this for a better compatibility with Darwin platforms.